docs/topics/serialization.txt

```diff
@@ -268,6 +268,15 @@ work::
                 return force_text(obj)
             return super(LazyEncoder, self).default(obj)
 
+By default, :class:`~django.core.serializers.json.DjangoJSONEncoder` is used
```

Comment:
The way our documentation build works is that a nonexistent target like this will fail the build, so you'll have to either remove it or add the heading. I think a subsection here that does that would be fine.